pandas/_libs/interval.pyx

```diff
@@ -381,17 +379,15 @@ cdef class Interval(IntervalMixin):
 
         left, right = self._repr_base()
         name = type(self).__name__
-        repr_str = '{name}({left!r}, {right!r}, closed={closed!r})'.format(
-            name=name, left=left, right=right, closed=self.closed)
+        repr_str = f'{name}({left}, {right}, closed={self.closed})'
```

Comment:
i think the problem is here, where you are dropping the "!r", which loses single-quotes around the affected values

Reply:
Oh, so it should be:
```f'{name}{left!r}, {right!r}......'  ```